### DISCUSSION TITLE
A chess tournament for EddieHub?

### DISCUSSION THREAD

**Discussion by adityaraute** [EddieHubCommunity/support#785] [Ideas]
I am recently very much hooked to chess. I think it is a game a lot of people will enjoy, if they don't already. 

Lichess.org is an open source chess website, and allows anyone to create a team and host a chess tournament among themselves. 

I can create the team, but will you guys be willing to participate?

**Comment by Vyvy-vi** [comment]
yah, I think we could try and host a few games :+1:

Moreover, I think `nhcarrigan` had a few ideas with scheduling free-for-all lobbies etc. for events. (that would be pretty epic)

**Comment by BOLT04** [comment]
Great idea @adityaraute 👍 💯. Could we use chess.com for these games or does it not have teams and tournaments?

**Reply by adityaraute** [reply]
We can technically, but I prefer `lichess`. Also, it's open source and we dig that here.

**Comment by eddiejaoude** [comment]
Sounds like fun, it is great if the community organises some collaborations - let me know how I can help. Also I will do something for BattleSnakes soon too 🤓

**Reply by Vyvy-vi** [reply]
how about we all also make chess engines and play against each other's engines :)

**Reply by adityaraute** [reply]
too difficult tbh

**Reply by eddiejaoude** [reply]
yeah that is hard, I struggle with batlesnakes lol

**Reply by Vyvy-vi** [reply]
hmm true.. How about a fun match where we have bots play with each other with random moves :joy:
